__tests__/components/Feed.test.js

```diff
@@ -0,0 +1,29 @@
+import React from 'react'
+import { shallow } from '../../enzyme';
+
+import Feed from '../../components/Feed';
+
+describe("Tests feed component", () => {
+  it("should render tweets list", () => {
+    const tweets = [
+      { id: 1, message: "Hello, testing", favorite: true },
+      { id: 2, message: "Hello, world", favorite: true }
+    ];
+
+    const wrapper = shallow(<Feed tweets={tweets} loading={true} />)
+    
+    expect(wrapper.exists()).toBeTruthy();
+    expect(wrapper.find("div.tweetDiv")).toHaveLength(tweets.length);  
```

Comment:
For these case of tests, you can use test of snapshot

```js
import React from 'react';
import Link from '../Link.react';
import renderer from 'react-test-renderer';

it('Link changes the class when hovered', () => {
  const wrapper = renderer.create(
    <Link page="http://www.facebook.com">Facebook</Link>,
  );
  expect(wrapper.toJSON()).toMatchSnapshot();
});
```

https://www.npmjs.com/package/react-test-renderer

Reply:
I gave snapshot a try on the feed tests. I'm guessing when i should rather snapshot or  compare manually *thinking_emoji*

Reply:
Yes, please, always pay attention before to update the snapshots